tests/unit/array/mixins/test_embed.py

```diff
@@ -40,10 +43,11 @@
 
 
 @pytest.mark.parametrize('framework', ['onnx', 'keras', 'pytorch', 'paddle'])
-@pytest.mark.parametrize('da', [DocumentArray])
+# @pytest.mark.parametrize('da', [DocumentArray, DocumentArraySqlite])
+@pytest.mark.parametrize('da', [DocumentArraySqlite])
 @pytest.mark.parametrize('N', [2, 1000])
 @pytest.mark.parametrize('batch_size', [1, 256])
-@pytest.mark.parametrize('to_numpy', [True, False])
+@pytest.mark.parametrize('to_numpy', [True])
```

Comment:
```suggestion
@pytest.mark.parametrize('to_numpy', [True, False])
```
just so that we don't forget